src/test/java/jenkins/plugins/hipchat/utils/BuildUtilsTest.java

```diff
@@ -129,12 +133,12 @@ public void collectedParametersContainDurationFromBuild() throws Exception {
     public void collectedParametersContainCorrectUrl() throws Exception {
         setupMocks();
 
-        given(jenkins.getRootUrl()).willReturn("http://localhost:8080/jenkins");
-        given(run.getUrl()).willReturn("/job/test%20project/1/");
+        given(jenkins.getRootUrl()).willReturn("http://localhost:" + rule.getLocalPort() + "/jenkins/");
+        given(run.getUrl()).willReturn("job/test%20project/1/");
```

Comment:
This looks correct as far as I can tell, the getRootURL method in Jenkins calls Util.ensureEndsWith(url,"/"); ,so most likely the test was incorrectly set up previously.